README.md

```
@@ -0,0 +1,176 @@
+# CodeReady Toolchain E2E Tests
+
+[![Go Report Card](https://goreportcard.com/badge/github.com/codeready-toolchain/toolchain-e2e)](https://goreportcard.com/report/github.com/codeready-toolchain/toolchain-e2e)
+[![GoDoc](https://godoc.org/github.com/codeready-toolchain/toolchain-e2e?status.png)](https://godoc.org/github.com/codeready-toolchain/toolchain-e2e)
+
+This repo contains e2e tests for [host](https://github.com/codeready-toolchain/host-operator) and [member](https://github.com/codeready-toolchain/member-operator) operators of CodeReady Toolchain.
+
+## Build
+
+Requires Go version 1.26.x (1.26.5 or higher) - download for your development environment [here](https://golang.org/dl/).
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)